### ISSUE TITLE
Change theme and language picker buttons are not accessible by keyboard

### ISSUE THREAD

**Issue by ShubhamOulkar** [expressjs/expressjs.com#1830]
![Image](https://github.com/user-attachments/assets/19556ba1-ecc7-4759-a42b-33d48f639ea5)
